example/package.json

```diff
@@ -5,11 +5,12 @@
     "dependencies": {
         "express": "^4.15.2",
         "monaco-editor-core": "^0.17.0",
-        "monaco-languageclient": "^0.10.0",
+        "monaco-languageclient": "0.13.0",
         "normalize-url": "^2.0.1",
         "reconnecting-websocket": "^3.2.2",
         "typescript-language-server": "0.5.1",
-        "vscode-ws-jsonrpc": "^0.1.1",
+        "vscode-languageserver-protocol": "3.15.3",
```

Comment:
Please take a look at this line here carefully, beceuse I am not sure why I needed to add this. :)

Let me explain, 
I added `"vscode-languageserver-protocol": "3.15.3"` in the `example/package.json` file to avoid. this error:
```
[compile] server/src/ts-protocol.ts(14,45): error TS2558: Expected 3 type arguments, but got 4.
```
First I tried to fix the issue in `server/src/ts-protocol.ts`, by providing 3 arguments instead of 4.
```diff
-    export const type = new lsp.RequestType<lsp.TextDocumentPositionParams, any, void, void>("_typescript.rename");
+    export const type = new lsp.RequestType<lsp.TextDocumentPositionParams, any, void>("_typescript.rename");

```
But doing that cause a lot of other typescript errors in the `/server` folder.

Somehow the `vscode-languageserver-protocol` in the /servers folder is different from the `vscode-languageserver-protocol` in the examples folder.

So by adding `"vscode-languageserver-protocol": "3.15.3"` I tell the `/example` folder to use a compatible version `vscode-languageserver-protocol` package and the `Expected 3 type arguments, but got 4.` was fixed by that.

Is this ok? :)